### PR TITLE
[MOB-2036] JWT callback test

### PR DESCRIPTION
**Pull request by roninopf**

### CODE REVIEW

File: tests/swift-sdk-swift-tests/AuthTests.swift [hunk 1]
```diff
@@ -217,4 +217,41 @@ class AuthTests: XCTestCase {
         XCTAssertEqual(internalAPI.userId, AuthTests.userId)
         XCTAssertEqual(internalAPI.auth.authToken, newAuthToken)
     }
+    
+    func testAuthFailureDelegateCall() {
+        let expectation1 = expectation(description: "")
```
Comment (tapashmajumder) [inline tests/swift-sdk-swift-tests/AuthTests.swift:222]:
```suggestion
        let expectation1 = expectation(description: #function)
```

Comment (tapashmajumder) [reply tests/swift-sdk-swift-tests/AuthTests.swift:222]: Minor suggestion. Not needed for +1.

**Review by tapashmajumder — APPROVED** [review]
+1
Please note other comment above.

File: tests/swift-sdk-swift-tests/AuthTests.swift (hunk 1 above)
Comment (roninopf) [reply tests/swift-sdk-swift-tests/AuthTests.swift]: Whoops. Added.

**Pull request merged**